### PR TITLE
added userChrome loader instructions

### PR DESCRIPTION
**Pull request by TheBigWazz**

### CODE REVIEW

**Comment by greeeen-dev** [comment]
Might be better to create some example CSS files to load so the loader isn't 100% comments

**Comment by TheBigWazz** [comment]
added examples in the readme screenshots, the loader file also has placeholder examples now too.

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+@import "userchrome-loader/Cohesion/Cohesion.css";
+@import "userchrome-loadernatsumi/config.css";
+@import "userchrome-loadernatsumi/preload.css";
+@import "userchrome-loadernatsumi/patches.css";
+@import "userchrome-loadernatsumi/base-ui.css";
+@import "userchrome-loadernatsumi/tab-groups.css";
+@import "userchrome-loadernatsumi/horizontal-tabs.css";
+@import "userchrome-loadernatsumi/natsumi-urlbar.css";
+@import "userchrome-loadernatsumi/natsumi-loading.css";
+@import "userchrome-loadernatsumi/ui-tweaks.css";
```
Comment (greeeen-dev) [inline Readme.md]: Missing some slashes here
```suggestion
@import "userchrome-loader/natsumi/config.css";
@import "userchrome-loader/natsumi/preload.css";
@import "userchrome-loader/natsumi/patches.css";
@import "userchrome-loader/natsumi/base-ui.css";
@import "userchrome-loader/natsumi/tab-groups.css";
@import "userchrome-loader/natsumi/horizontal-tabs.css";
@import "userchrome-loader/natsumi/natsumi-urlbar.css";
@import "userchrome-loader/natsumi/natsumi-loading.css";
@import "userchrome-loader/natsumi/ui-tweaks.css";
```

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# userChrome Loader
+This is a template intended to make it easier for end-users to manually load and manage custom CSS.
+
+## Setup Instructions
+**Step 1.** In your **chrome** folder, save the **userchrome-loader** folder from this repo.
```
Comment (greeeen-dev) [inline Readme.md]: Might be better to use standard numbered lists here so we can take advantage of the numbered list markdown.

e.g.
1. do this
2. do that

File: userchrome-loader/loader.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    Add an @import rule for each new file you create in this file. */
+    
+/* --- Examples --- */
+/* @import "userchrome-loader/example.css"; */
+/* @import "userchrome-loader/new-folder/example2.css"; */
+
+/*--------------------------------------------------------------------*/
+
+/* === Load custom files === */
```
Comment (greeeen-dev) [inline userchrome-loader/loader.css]: Can you make some example files in userchrome-loader/modules (make a new folder) so we can add import statements for those here? Would rather not have the loader example be 100% comments

**Review by greeeen-dev — CHANGES_REQUESTED** [review]

File: Readme.md (hunk 1 above)
Comment (TheBigWazz) [reply Readme.md]: resolved in a commit as you were reviewing

Comment (greeeen-dev) [reply Readme.md]: lmao

File: Readme.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+## Setup Instructions
+1. In your **chrome** folder, save the **userchrome-loader** folder from this repo.
+   
+2. If you already have custom CSS in your **userChrome.css** file, back it up by copying the contents to a new CSS file indside the **userchrome-loader** folder.
+  
+  - If you don't have any custom CSS in your userChrome.css file, skip this step.
+
+3. Copy the **userChrome.css** file from this repo into your own userChrome.css file. 
+
+  - This should be the only content in your userChrome.css file.
```
Comment (greeeen-dev) [inline Readme.md]: This should use three spaces, not two. In its current state it breaks the formatting.
```suggestion
   - This should be the only content in your userChrome.css file.
```

File: Readme.md
Comment (greeeen-dev) [inline Readme.md:1]: Can we rename this to README.md so we're consistent with repositories that use all capitals (except for the format)?

**Review by greeeen-dev — CHANGES_REQUESTED** [review]
Everything else looks fine, just two minor issues

File: Readme.md
Comment (greeeen-dev) [reply Readme.md:1]: Renamed this myself so we can merge it quickly

**Review by greeeen-dev — APPROVED** [review]
All good now, merging this

**Pull request merged**